rmgpy/rmg/main.py

```diff
@@ -550,10 +557,10 @@ def initialize(self, **kwargs):
         self.load_database()
 
         for spec in self.initial_species:
-            self.reaction_model.add_species_to_edge(spec)
+            self.reaction_model.add_species_to_edge(spec, requires_rms=requires_rms)
 
         for reaction_system in self.reaction_systems:
-            if isinstance(reaction_system, Reactor):
+            if not NO_JULIA and isinstance(reaction_system, RMSReactor):
```

Comment:
What happens if there is no julia and the user specify to use an RMSReactor? Should we raise an error for that? Or have we already raised an error for that? If we have, I think we don't need to check NO_JULIA here again?

Reply:
You are right - in the lines above I have actually left a comment that we would have already failed. So I will remove this part of the check.

Reply:
I think there are other instances where we check both `not NO_JULIA` and `isinstance(reaction_system, RMSReactor)`, but if `NO_JULIA` and `isinstance(reaction_system, RMSReactor)` that should have already failed, I don't think we need to check them again. Could you go through these cases and remove the unnecessary checks?

Reply:
Good point, just got them all. We can rely only on `requires_rms`, though `NO_JULIA` is still used for better errors and in the initialization of the model (which must stay, since it doesn't know anything the reactors being used).